2022-07_sprint/sprint_parties/contributors.md

```diff
@@ -99,16 +99,4 @@ São Paulo, São Paulo (Brazil)
 [{fab}`github`](https://github.com/vitaliset)
 [{fab}`linkedin-in`](https://www.linkedin.com/in/carlo-lemos)
 :::
```

Comment:
```suggestion
:::{grid-item}

<img alt="Image for contributor template" src="../../_static/contributors/blank_woman.jpg" class="rounded-circle">

Mariya

(USA)

[{fab}`github`](https://github.com/mariyayb)
[{fab}`linkedin-in`](https://www.linkedin.com/in/mariya-b/)
:::
:::{grid-item}

<img alt="Image for contributor template" src="../../_static/contributors/blank_man.jpg" class="rounded-circle">

Elijah Appiah

Winneba, Central Region (Ghana)

[{fab}`github`](https://github.com/elijah-appiah)
[{fab}`linkedin-in`](https://www.linkedin.com/in/appiah-elijah-383231123/)
:::
```

Reply:
Let's try this.

Reply:
I'm confused what happened here.  The first several of your commits seems fine, but then the last 2 just remove the 2 contributors I added.  What was not working?

Reply:
@cluhmann I couldn't figure out why it wasn't working.
The code looked the same as my PR. ???

Reply:
What exactly "wasn't working"?  It's not clear from the commits.

Reply:
I have no idea why it wasn't working.  I only know that the docs weren't being built, so I was trying to trouble shoot.  My PR did seem to work, in terms of building the docs.

Reply:
If you open the preview,  the version witcher menu has a "construccions" (in catalan for me, in english I'd expect constructions or builds). From there you can check the logs and search for warnings and errors.

The builds page: https://readthedocs.org/projects/pymc-data-umbrella/builds/

This is PR 112, if I open the first log for this ans search for warnings and errors I get image not readable. That means either the path/name/extension are wrong or the file is corrupted (not the case here). Log: https://readthedocs.org/projects/pymc-data-umbrella/builds/17279758/

Reply:
The image extension was wrong (PNG--> JPG). 
But, it still did not solve the problem.

I think I might know the problem. In my PR, I ended up renaming the image, so it would match.  
But, it wasn't renamed in @cluhmann's PR.
Maybe that was it.

Reply:
Strange.  Closing because this has been superseded by #114 .

Reply:
I am so might be messing up, but I don't see any commit updating the extension in the image. In any case, the build logs of each commit are available, you can also check if this message is solved and a new one appears